next/README.md

```diff
@@ -1,73 +1,54 @@
-
-**Multiple Contents allows storing more than one content model per tile**. This enables storing more than one content in a single region of space. Contents can be organized in several ways (layers, arbitrary groupings), especially when combined with group metadata.
-
-<img src="figures/multiple-contents.png" width="600px" alt="Tileset with multiple contents">
-
-## Specifications
+# 3D Tiles Next
 
-### [`3DTILES_content_gltf`](../extensions/3DTILES_content_gltf) (3D Tiles 1.0 extension)
+## Overview
 
-This extension simply allows glTF models (`.gltf` `.glb`) directly in `tile.content`. This allows for easier integration with software that may produce glTF files, but not 3D Tiles 1.0 formats (such as B3DM).
+**3D Tiles Next** is a set of new capabilities for the future of 3D Tiles. Provided as extensions to the [3D Tiles 1.0](../specification) specification, these features may be incorporated into 3D Tiles 2.0 in the future.
 
-### [`3DTILES_implicit_tiling`](../extensions/3DTILES_implicit_tiling) (3D Tiles 1.0 extension)
+- [Tile Content](#tile-content): glTF 2.0 assets may be used directly as tile content, without intermediate formats, improving interoperability with 3D content and tooling ecosystems. Tiles may reference multiple contents — for organization, styling, and filtering — and contents may be collected into groups similar to map layers in mapping applications.
+- [Implicit Tiling](#implicit-tiling): Common subdivision schemes and spatial index patterns may be declared without listing bounding volumes exhaustively. Reduces tileset size, and enables new optimizations including faster traversal, raycasting, random access, and spatial queries.
+- [Metadata](#metadata): Metadata in 3D Tiles gains more expressiveness and flexibility, with a well-defined type system, new encoding options (e.g. JSON or binary), and a range of granularity options. Metadata may be associated with high-level objects like tilesets, tiles, or tile content groups, or with individual vertices and texels on glTF 2.0 geometry.
 
-This extension introduces a more compact representation of quadtree and octree tilesets. It defines subtree files, which are binary files that represent which tiles and contents exist for a fixed-sized portion of the tree. This is more compact than explicitly listing bounding volumes for every tile.
+## Tile Content
 
-Subtree files are designed to facilitate fast tree traversals, as each tile can be uniquely identified by its tile coordinates. It also makes it possible to use alternate traversal algorithms, or perform ray tracing.
+_**Overview:** glTF 2.0 assets may be used directly as tile content, without intermediate formats, improving interoperability with 3D content and tooling ecosystems. Tiles may reference multiple contents — for organization, styling, and filtering — and contents may be collected into groups similar to map layers in mapping applications._
```

Comment:
Ok – if this level of detail is otherwise good (i.e. we don't expect it to _become_ a particularly long document) then let's just remove the three bullets at the top. Note these are also repeated in the main README.md, but that seems more justifiable.